# Genomics of the outbreak of Chikungunya virus in Cambodia 2020

This repository analyzes Chikungunya virus (CHIKV) genomes circulating in Cambodia 2020 (n=73) using [Nextstrain](https://nextstrain.org/).
The build was constructed using a total of 775 CHIKV full-length genomes encompassing the main 4 CHIKV lineages: West African (WA), Asian, East Central South African (ECSA) and Indian Ocean Lineage (IOL).The coloring option subclade displays the distribution of subclades within the main CHIKV lineages and in particular for the IOL lineage we show the distribution of AA substitutions such as the E1:A226V, E1:K211E and E2:V264. These substitutions have been widely identified and thus believed to be potentially biological relevant mutations.

This work is ongoing, and is performed in collaboration with [Institut Pasteur du Cambodge](https://www.pasteur-kh.org/) (contact: Dr Veasna Duong). Please do not hesitate to contact us if this could be relevant to your work!

Phylogeny visualized at https://nextstrain.org/community/Simon-LoriereLab/ChikungunyaCambodia2020@main

Phylogeny with focus on CHIKV from Cambodia 2020 can be visualized at https://nextstrain.org/community/Simon-LoriereLab/ChikungunyaCambodia2020@main/subset

Narratives available at https://nextstrain.org/community/narratives/Simon-LoriereLab/ChikungunyaCambodia2020@main


## Primers scheme

We provide here a short amplicon scheme developped to help sequence low concentration or partially degraded samples corresponding to the CHIKV IOL sublineage circulating in SouthEast Asia in recent years.
The primers scheme was built using [Primal Scheme](https://primalscheme.com/) and we followed the [PrimalSeq](https://www.protocols.io/view/primalseq-generation-of-tiled-virus-amplicons-for-bez7jf9n) protocol for the generation of tiled virus amplicons.
